doc/source/ray-air/examples/torch_incremental_learning.ipynb

```diff
@@ -0,0 +1,1703 @@
+        "        for permutation in self.permutations:\n",
+        "            permuted_mnist_dataset = self.random_permute_dataset(\n",
+        "                self.train_mnist_dataset, permutation\n",
+        "            )\n",
+        "            yield permuted_mnist_dataset\n",
+        "\n",
+        "    def generate_test_stream(self) -> Iterator[ray.data.Dataset]:\n",
+        "        for permutation in self.permutations:\n",
+        "            mnist_dataset = get_mnist_dataset(train=False)\n",
+        "            permuted_mnist_dataset = self.random_permute_dataset(\n",
+        "                self.test_mnist_dataset, permutation\n",
+        "            )\n",
+        "            yield permuted_mnist_dataset\n",
+        "\n",
+        "    def generate_test_samples(self, num_samples: int = 10) -> List[np.ndarray]:\n",
+        "      \"\"\"Generates num_samples permuted MNIST images.\"\"\"\n",
+        "      random_permutation =  random.choice(self.permutations)\n",
+        "      return self.random_permute_dataset(self.test_mnist_dataset.random_shuffle().limit(num_samples), random_permutation).to_pandas()[\"image\"].to_list()\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HDGHgtb699kd"
+      },
+      "source": [
+        "# Step 4: Define the logic for Training and Inference/Prediction\n",
+        "\n",
+        "Now that we can get an Iterator over Ray Datasets, we can incrementally train our model in a data parallel fashion via Ray Train, while incrementally deploying our model via Ray Serve. Let's define some helper functions to allow us to do this!\n",
+        "\n",
+        "If you are not familiar with data parallel training, it is a form of distributed training strategies, where we have multiple model replicas, and each replica trains on a different batch of data. After each batch, the gradients are synchronized across the replicas. This effecitively allows us to train on more data in a shorter amount of time."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SBWxP1sP-G-o"
+      },
+      "source": [
+        "## 4a: Define our training logic for each Data Parallel worker\n",
+        "\n",
+        "The first thing we need to do is to define the training loop that will be run on each training worker. \n",
+        "\n",
+        "The training loop takes in a `config` Dict as an argument that we can use to pass in any configurations for training.\n",
+        "\n",
+        "This is just standard PyTorch training, with the difference being that we can leverage [Ray Train's utility functions](https://docs.ray.io/en/master/train/api.html#training-function-utilities):\n",
+        "- `ray.train.torch.prepare_model(...)`: This will the prepare model for distributed training by wrapping it in PyTorch `DistributedDataParallel` and moving it to the correct accelerator device.\n",
```

Comment:
```suggestion
        "- `ray.train.torch.prepare_model(...)`: This will prepare the model for distributed training by wrapping it in PyTorch `DistributedDataParallel` and moving it to the correct accelerator device.\n",
```